backend/infrahub/core/node/create.py

```diff
@@ -136,7 +164,7 @@ async def handle_template_relationships(
             )
 
 
-async def get_profile_ids(db: InfrahubDatabase, obj: Node) -> set[str]:
+async def get_profile_ids(db: InfrahubDatabase, obj: Node | CoreObjectTemplate) -> set[str]:
```

Comment:
A `CoreObjectTemplate` is somewhat a `Node` so I'm not sure the typing need to be changed here. But that's minor comment.